src/sagemaker/transformer.py

```diff
@@ -194,6 +197,11 @@ def transform(
                 (default: ``True``).
             logs (bool): Whether to show the logs produced by the job.
                 Only meaningful when wait is ``True`` (default: ``True``).
+            kms_key (str): The ARN of the KMS key that is used to encrypt the
```

Comment:
Don't see the kms_key in the .transform() signature. Why we add it here in doc string?

Reply:
Copy/Paste error. Removed it.